<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>qcd_jk_report</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">qcd_jk_report</h1>
<p class='attribute'><strong>测试人员 : </strong> keen</p>
<p class='attribute'><strong>开始时间 : </strong> 2019-07-09 18:02:51</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:31.753685</p>
<p class='attribute'><strong>测试结果 : </strong> 共 53，通过 39，错误 14，通过率= 73.58%</p>

<p class='description'>前程贷项目接口测试第1份报告</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 73.58% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 0 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 39 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 53 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_sendMCode.SendMCodeTestCase</td>
    <td class="text-center">53</td>
    <td class="text-center">39</td>
    <td class="text-center">0</td>
    <td class="text-center">14</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',53)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='pt1_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_01: 正常发送（手机号段130）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse in" align="left">
    <pre>
    
pt1_1: 请求数据: {'client_ip': '203.178.45.209', 'tmpl_id': '1', 'mobile': '13023586010'}
请求数据--&gt; {'client_ip': '203.178.45.209', 'tmpl_id': '1', 'mobile': '13023586010'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_02: 正常发送（手机号段131）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_2'>通过</button>
    <div id='div_pt1_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_2'>通过</button>
    <div id='div_pt1_2' class="collapse in" align="left">
    <pre>
    
pt1_2: 请求数据: {'client_ip':'105.75.27.35', 'tmpl_id':'1', 'mobile':'13134257153'}
请求数据--&gt; {'client_ip':'105.75.27.35', 'tmpl_id':'1', 'mobile':'13134257153'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_03: 正常发送（手机号段132）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_3'>通过</button>
    <div id='div_pt1_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_3'>通过</button>
    <div id='div_pt1_3' class="collapse in" align="left">
    <pre>
    
pt1_3: 请求数据: {'client_ip':'252.152.254.149', 'tmpl_id':'1', 'mobile':'13239345001'}
请求数据--&gt; {'client_ip':'252.152.254.149', 'tmpl_id':'1', 'mobile':'13239345001'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_04: 正常发送（手机号段133）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_4'>通过</button>
    <div id='div_pt1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_4'>通过</button>
    <div id='div_pt1_4' class="collapse in" align="left">
    <pre>
    
pt1_4: 请求数据: {'client_ip':'106.212.31.229', 'tmpl_id':'1', 'mobile':'13350461015'}
请求数据--&gt; {'client_ip':'106.212.31.229', 'tmpl_id':'1', 'mobile':'13350461015'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_5' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_05: 正常发送（手机号段134）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_5'>通过</button>
    <div id='div_pt1_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_5'>通过</button>
    <div id='div_pt1_5' class="collapse in" align="left">
    <pre>
    
pt1_5: 请求数据: {'client_ip':'10.82.189.138', 'tmpl_id':'1', 'mobile':'13417129977'}
请求数据--&gt; {'client_ip':'10.82.189.138', 'tmpl_id':'1', 'mobile':'13417129977'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_6' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_06: 正常发送（手机号段135）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_6'>通过</button>
    <div id='div_pt1_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_6'>通过</button>
    <div id='div_pt1_6' class="collapse in" align="left">
    <pre>
    
pt1_6: 请求数据: {'client_ip':'118.149.25.43', 'tmpl_id':'1', 'mobile':'13572845248'}
请求数据--&gt; {'client_ip':'118.149.25.43', 'tmpl_id':'1', 'mobile':'13572845248'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_07: 正常发送（手机号段136）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_7'>通过</button>
    <div id='div_pt1_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_7'>通过</button>
    <div id='div_pt1_7' class="collapse in" align="left">
    <pre>
    
pt1_7: 请求数据: {'client_ip':'96.27.124.246', 'tmpl_id':'1', 'mobile':'13659835207'}
请求数据--&gt; {'client_ip':'96.27.124.246', 'tmpl_id':'1', 'mobile':'13659835207'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_8' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_08: 正常发送（手机号段137）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_8'>通过</button>
    <div id='div_pt1_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_8'>通过</button>
    <div id='div_pt1_8' class="collapse in" align="left">
    <pre>
    
pt1_8: 请求数据: {'client_ip':'255.22.171.38', 'tmpl_id':'1', 'mobile':'13738501969'}
请求数据--&gt; {'client_ip':'255.22.171.38', 'tmpl_id':'1', 'mobile':'13738501969'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_9' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_09: 正常发送（手机号段138）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_9'>通过</button>
    <div id='div_pt1_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_9'>通过</button>
    <div id='div_pt1_9' class="collapse in" align="left">
    <pre>
    
pt1_9: 请求数据: {'client_ip':'67.241.121.123', 'tmpl_id':'1', 'mobile':'13836489934'}
请求数据--&gt; {'client_ip':'67.241.121.123', 'tmpl_id':'1', 'mobile':'13836489934'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_10' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_10: 正常发送（手机号段139）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_10'>通过</button>
    <div id='div_pt1_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_10'>通过</button>
    <div id='div_pt1_10' class="collapse in" align="left">
    <pre>
    
pt1_10: 请求数据: {'client_ip':'203.90.213.119', 'tmpl_id':'1', 'mobile':'13900611993'}
请求数据--&gt; {'client_ip':'203.90.213.119', 'tmpl_id':'1', 'mobile':'13900611993'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_11' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_11: 正常发送（手机号段145）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_11'>通过</button>
    <div id='div_pt1_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_11'>通过</button>
    <div id='div_pt1_11' class="collapse in" align="left">
    <pre>
    
pt1_11: 请求数据: {'client_ip':'205.50.223.41', 'tmpl_id':'1', 'mobile':'14573952602'}
请求数据--&gt; {'client_ip':'205.50.223.41', 'tmpl_id':'1', 'mobile':'14573952602'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_12' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_12: 正常发送（手机号段147）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_12'>通过</button>
    <div id='div_pt1_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_12'>通过</button>
    <div id='div_pt1_12' class="collapse in" align="left">
    <pre>
    
pt1_12: 请求数据: {'client_ip':'69.220.243.210', 'tmpl_id':'1', 'mobile':'14728855636'}
请求数据--&gt; {'client_ip':'69.220.243.210', 'tmpl_id':'1', 'mobile':'14728855636'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_13: 正常发送（手机号段149）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse in" align="left">
    <pre>
    
ft1_13: 请求数据: {'client_ip':'106.207.223.123', 'tmpl_id':'1', 'mobile':'14993772708'}
&lt;suds.sax.document.Document object at 0x000001DFC51D6C18&gt;
请求数据--&gt; {'client_ip':'106.207.223.123', 'tmpl_id':'1', 'mobile':'14993772708'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_14' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_14: 正常发送（手机号段150）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_14'>通过</button>
    <div id='div_pt1_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_14'>通过</button>
    <div id='div_pt1_14' class="collapse in" align="left">
    <pre>
    
pt1_14: 请求数据: {'client_ip':'161.51.255.41', 'tmpl_id':'1', 'mobile':'15079560980'}
请求数据--&gt; {'client_ip':'161.51.255.41', 'tmpl_id':'1', 'mobile':'15079560980'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_15' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_15: 正常发送（手机号段151）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_15'>通过</button>
    <div id='div_pt1_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_15'>通过</button>
    <div id='div_pt1_15' class="collapse in" align="left">
    <pre>
    
pt1_15: 请求数据: {'client_ip':'31.57.2.185', 'tmpl_id':'1', 'mobile':'15104062459'}
请求数据--&gt; {'client_ip':'31.57.2.185', 'tmpl_id':'1', 'mobile':'15104062459'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_16' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_16: 正常发送（手机号段152）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_16'>通过</button>
    <div id='div_pt1_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_16'>通过</button>
    <div id='div_pt1_16' class="collapse in" align="left">
    <pre>
    
pt1_16: 请求数据: {'client_ip':'115.233.156.70', 'tmpl_id':'1', 'mobile':'15258630577'}
请求数据--&gt; {'client_ip':'115.233.156.70', 'tmpl_id':'1', 'mobile':'15258630577'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_17' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_17: 正常发送（手机号段153）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_17'>通过</button>
    <div id='div_pt1_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_17'>通过</button>
    <div id='div_pt1_17' class="collapse in" align="left">
    <pre>
    
pt1_17: 请求数据: {'client_ip':'58.18.61.181', 'tmpl_id':'1', 'mobile':'15318987588'}
请求数据--&gt; {'client_ip':'58.18.61.181', 'tmpl_id':'1', 'mobile':'15318987588'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_18' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_18: 正常发送（手机号段155）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_18'>通过</button>
    <div id='div_pt1_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_18'>通过</button>
    <div id='div_pt1_18' class="collapse in" align="left">
    <pre>
    
pt1_18: 请求数据: {'client_ip':'63.90.15.64', 'tmpl_id':'1', 'mobile':'15579479677'}
请求数据--&gt; {'client_ip':'63.90.15.64', 'tmpl_id':'1', 'mobile':'15579479677'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_19' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_19: 正常发送（手机号段156）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_19' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_19'>通过</button>
    <div id='div_pt1_19' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_19' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_19'>通过</button>
    <div id='div_pt1_19' class="collapse in" align="left">
    <pre>
    
pt1_19: 请求数据: {'client_ip':'161.228.220.3', 'tmpl_id':'1', 'mobile':'15610058314'}
请求数据--&gt; {'client_ip':'161.228.220.3', 'tmpl_id':'1', 'mobile':'15610058314'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_20' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_20: 正常发送（手机号段157）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_20' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_20'>通过</button>
    <div id='div_pt1_20' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_20' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_20'>通过</button>
    <div id='div_pt1_20' class="collapse in" align="left">
    <pre>
    
pt1_20: 请求数据: {'client_ip':'100.68.125.126', 'tmpl_id':'1', 'mobile':'15736808154'}
请求数据--&gt; {'client_ip':'100.68.125.126', 'tmpl_id':'1', 'mobile':'15736808154'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_21' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_21: 正常发送（手机号段158）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_21' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_21'>通过</button>
    <div id='div_pt1_21' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_21' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_21'>通过</button>
    <div id='div_pt1_21' class="collapse in" align="left">
    <pre>
    
pt1_21: 请求数据: {'client_ip':'151.21.143.211', 'tmpl_id':'1', 'mobile':'15877196080'}
请求数据--&gt; {'client_ip':'151.21.143.211', 'tmpl_id':'1', 'mobile':'15877196080'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_22' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_22: 正常发送（手机号段159）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_22' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_22'>通过</button>
    <div id='div_pt1_22' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_22' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_22'>通过</button>
    <div id='div_pt1_22' class="collapse in" align="left">
    <pre>
    
pt1_22: 请求数据: {'client_ip':'172.30.194.189', 'tmpl_id':'1', 'mobile':'15907786797'}
请求数据--&gt; {'client_ip':'172.30.194.189', 'tmpl_id':'1', 'mobile':'15907786797'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_23' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_23: 正常发送（手机号段166）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse in" align="left">
    <pre>
    
ft1_23: 请求数据: {'client_ip':'182.69.130.183', 'tmpl_id':'1', 'mobile':'16605578388'}
&lt;suds.sax.document.Document object at 0x000001DFC539C0F0&gt;
请求数据--&gt; {'client_ip':'182.69.130.183', 'tmpl_id':'1', 'mobile':'16605578388'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_24' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_24: 正常发送（手机号段170）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_24' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_24'>通过</button>
    <div id='div_pt1_24' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_24' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_24'>通过</button>
    <div id='div_pt1_24' class="collapse in" align="left">
    <pre>
    
pt1_24: 请求数据: {'client_ip':'153.105.195.221', 'tmpl_id':'1', 'mobile':'17097746377'}
请求数据--&gt; {'client_ip':'153.105.195.221', 'tmpl_id':'1', 'mobile':'17097746377'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_25' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_25: 正常发送（手机号段171）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse in" align="left">
    <pre>
    
ft1_25: 请求数据: {'client_ip':'158.2.129.200', 'tmpl_id':'1', 'mobile':'17141914629'}
&lt;suds.sax.document.Document object at 0x000001DFC507D6A0&gt;
请求数据--&gt; {'client_ip':'158.2.129.200', 'tmpl_id':'1', 'mobile':'17141914629'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_26' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_26: 正常发送（手机号段172）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse in" align="left">
    <pre>
    
ft1_26: 请求数据: {'client_ip':'4.90.242.211', 'tmpl_id':'1', 'mobile':'17229951457'}
&lt;suds.sax.document.Document object at 0x000001DFC535B8D0&gt;
请求数据--&gt; {'client_ip':'4.90.242.211', 'tmpl_id':'1', 'mobile':'17229951457'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_27' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_27: 正常发送（手机号段173）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse in" align="left">
    <pre>
    
ft1_27: 请求数据: {'client_ip':'215.24.116.200', 'tmpl_id':'1', 'mobile':'17318519140'}
&lt;suds.sax.document.Document object at 0x000001DFC5698710&gt;
请求数据--&gt; {'client_ip':'215.24.116.200', 'tmpl_id':'1', 'mobile':'17318519140'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_28' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_28: 正常发送（手机号段175）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse in" align="left">
    <pre>
    
ft1_28: 请求数据: {'client_ip':'125.163.118.7', 'tmpl_id':'1', 'mobile':'17533173996'}
&lt;suds.sax.document.Document object at 0x000001DFC5460390&gt;
请求数据--&gt; {'client_ip':'125.163.118.7', 'tmpl_id':'1', 'mobile':'17533173996'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_29' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_29: 正常发送（手机号段176）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_29' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_29'>通过</button>
    <div id='div_pt1_29' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_29' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_29'>通过</button>
    <div id='div_pt1_29' class="collapse in" align="left">
    <pre>
    
pt1_29: 请求数据: {'client_ip':'126.150.133.255', 'tmpl_id':'1', 'mobile':'17605121133'}
请求数据--&gt; {'client_ip':'126.150.133.255', 'tmpl_id':'1', 'mobile':'17605121133'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_30' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_30: 正常发送（手机号段177）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_30' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_30'>通过</button>
    <div id='div_pt1_30' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_30' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_30'>通过</button>
    <div id='div_pt1_30' class="collapse in" align="left">
    <pre>
    
pt1_30: 请求数据: {'client_ip':'22.81.137.137', 'tmpl_id':'1', 'mobile':'17784517324'}
请求数据--&gt; {'client_ip':'22.81.137.137', 'tmpl_id':'1', 'mobile':'17784517324'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_31' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_31: 正常发送（手机号段178）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_31' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_31'>通过</button>
    <div id='div_pt1_31' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_31' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_31'>通过</button>
    <div id='div_pt1_31' class="collapse in" align="left">
    <pre>
    
pt1_31: 请求数据: {'client_ip':'236.36.62.107', 'tmpl_id':'1', 'mobile':'17839690540'}
请求数据--&gt; {'client_ip':'236.36.62.107', 'tmpl_id':'1', 'mobile':'17839690540'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_32' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_32: 正常发送（手机号段180）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_32' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_32'>通过</button>
    <div id='div_pt1_32' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_32' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_32'>通过</button>
    <div id='div_pt1_32' class="collapse in" align="left">
    <pre>
    
pt1_32: 请求数据: {'client_ip':'214.222.189.227', 'tmpl_id':'1', 'mobile':'18089265252'}
请求数据--&gt; {'client_ip':'214.222.189.227', 'tmpl_id':'1', 'mobile':'18089265252'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_33' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_33: 正常发送（手机号段181）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_33' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_33'>通过</button>
    <div id='div_pt1_33' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_33' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_33'>通过</button>
    <div id='div_pt1_33' class="collapse in" align="left">
    <pre>
    
pt1_33: 请求数据: {'client_ip':'96.226.116.78', 'tmpl_id':'1', 'mobile':'18196503097'}
请求数据--&gt; {'client_ip':'96.226.116.78', 'tmpl_id':'1', 'mobile':'18196503097'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_34' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_34: 正常发送（手机号段182）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_34' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_34'>通过</button>
    <div id='div_pt1_34' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_34' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_34'>通过</button>
    <div id='div_pt1_34' class="collapse in" align="left">
    <pre>
    
pt1_34: 请求数据: {'client_ip':'106.79.172.133', 'tmpl_id':'1', 'mobile':'18254692361'}
请求数据--&gt; {'client_ip':'106.79.172.133', 'tmpl_id':'1', 'mobile':'18254692361'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_35' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_35: 正常发送（手机号段183）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_35' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_35'>通过</button>
    <div id='div_pt1_35' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_35' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_35'>通过</button>
    <div id='div_pt1_35' class="collapse in" align="left">
    <pre>
    
pt1_35: 请求数据: {'client_ip':'86.51.225.21', 'tmpl_id':'1', 'mobile':'18371776037'}
请求数据--&gt; {'client_ip':'86.51.225.21', 'tmpl_id':'1', 'mobile':'18371776037'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_36' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_36: 正常发送（手机号段184）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_36' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_36'>通过</button>
    <div id='div_pt1_36' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_36' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_36'>通过</button>
    <div id='div_pt1_36' class="collapse in" align="left">
    <pre>
    
pt1_36: 请求数据: {'client_ip':'188.74.75.39', 'tmpl_id':'1', 'mobile':'18497183052'}
请求数据--&gt; {'client_ip':'188.74.75.39', 'tmpl_id':'1', 'mobile':'18497183052'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_37' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_37: 正常发送（手机号段185）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_37' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_37'>通过</button>
    <div id='div_pt1_37' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_37' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_37'>通过</button>
    <div id='div_pt1_37' class="collapse in" align="left">
    <pre>
    
pt1_37: 请求数据: {'client_ip':'100.122.189.64', 'tmpl_id':'1', 'mobile':'18533396609'}
请求数据--&gt; {'client_ip':'100.122.189.64', 'tmpl_id':'1', 'mobile':'18533396609'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_38' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_38: 正常发送（手机号段186）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_38' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_38'>通过</button>
    <div id='div_pt1_38' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_38' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_38'>通过</button>
    <div id='div_pt1_38' class="collapse in" align="left">
    <pre>
    
pt1_38: 请求数据: {'client_ip':'153.216.127.197', 'tmpl_id':'1', 'mobile':'18605495391'}
请求数据--&gt; {'client_ip':'153.216.127.197', 'tmpl_id':'1', 'mobile':'18605495391'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_39' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_39: 正常发送（手机号段187）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_39' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_39'>通过</button>
    <div id='div_pt1_39' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_39' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_39'>通过</button>
    <div id='div_pt1_39' class="collapse in" align="left">
    <pre>
    
pt1_39: 请求数据: {'client_ip':'82.192.131.106', 'tmpl_id':'1', 'mobile':'18753004951'}
请求数据--&gt; {'client_ip':'82.192.131.106', 'tmpl_id':'1', 'mobile':'18753004951'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_40' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_40: 正常发送（手机号段188）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_40' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_40'>通过</button>
    <div id='div_pt1_40' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_40' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_40'>通过</button>
    <div id='div_pt1_40' class="collapse in" align="left">
    <pre>
    
pt1_40: 请求数据: {'client_ip':'112.14.230.185', 'tmpl_id':'1', 'mobile':'18855806216'}
请求数据--&gt; {'client_ip':'112.14.230.185', 'tmpl_id':'1', 'mobile':'18855806216'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_41' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_41: 正常发送（手机号段189）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_41' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_41'>通过</button>
    <div id='div_pt1_41' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_41' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_41'>通过</button>
    <div id='div_pt1_41' class="collapse in" align="left">
    <pre>
    
pt1_41: 请求数据: {'client_ip':'26.126.205.86', 'tmpl_id':'1', 'mobile':'18926802088'}
请求数据--&gt; {'client_ip':'26.126.205.86', 'tmpl_id':'1', 'mobile':'18926802088'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_42' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_42: 正常发送（手机号段198）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_42' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_42'>错误</button>
    <div id='div_ft1_42' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_42' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_42'>错误</button>
    <div id='div_ft1_42' class="collapse in" align="left">
    <pre>
    
ft1_42: 请求数据: {'client_ip':'159.173.159.49', 'tmpl_id':'1', 'mobile':'19888798193'}
&lt;suds.sax.document.Document object at 0x000001DFC54A6DD8&gt;
请求数据--&gt; {'client_ip':'159.173.159.49', 'tmpl_id':'1', 'mobile':'19888798193'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_43' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_43: 正常发送（手机号段199）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_43' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_43'>错误</button>
    <div id='div_ft1_43' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_43' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_43'>错误</button>
    <div id='div_ft1_43' class="collapse in" align="left">
    <pre>
    
ft1_43: 请求数据: {'client_ip':'137.99.91.125', 'tmpl_id':'1', 'mobile':'19961998837'}
&lt;suds.sax.document.Document object at 0x000001DFC50A8D68&gt;
请求数据--&gt; {'client_ip':'137.99.91.125', 'tmpl_id':'1', 'mobile':'19961998837'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 86, in test_send_m_code
    self.assertEqual(eval(case.expected_data), response)
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: {'retCode': '0', 'retInfo': 'ok'} != "{'faultcode': soap:Server, 'faultstring': 手机号码错误}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_44' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_44: ip有负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_44' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_44'>错误</button>
    <div id='div_ft1_44' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_44' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_44'>错误</button>
    <div id='div_ft1_44' class="collapse in" align="left">
    <pre>
    
ft1_44: 请求数据: {'client_ip':'-100.1.1.1', 'tmpl_id':'1', 'mobile':'13357042152'}
请求数据--&gt; {'client_ip':'-100.1.1.1', 'tmpl_id':'1', 'mobile':'13357042152'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': ip非法}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring': ip非法}" != "{'retCode': 0, 'retInfo': 'ok'}"
- {'faultcode': soap:Server, 'faultstring': ip非法}
+ {'retCode': 0, 'retInfo': 'ok'}


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_45' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_45: ip只有1位</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_45' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_45'>错误</button>
    <div id='div_ft1_45' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_45' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_45'>错误</button>
    <div id='div_ft1_45' class="collapse in" align="left">
    <pre>
    
ft1_45: 请求数据: {'client_ip':'2', 'tmpl_id':'1', 'mobile':'13315260003'}
请求数据--&gt; {'client_ip':'2', 'tmpl_id':'1', 'mobile':'13315260003'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': ip非法}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring': ip非法}" != "{'retCode': 0, 'retInfo': 'ok'}"
- {'faultcode': soap:Server, 'faultstring': ip非法}
+ {'retCode': 0, 'retInfo': 'ok'}


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_46' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_46: ip超过最大地址</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_46' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_46'>错误</button>
    <div id='div_ft1_46' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_46' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_46'>错误</button>
    <div id='div_ft1_46' class="collapse in" align="left">
    <pre>
    
ft1_46: 请求数据: {'client_ip':'300.59.500.1000', 'tmpl_id':1, 'mobile':'13346655259'}
请求数据--&gt; {'client_ip':'300.59.500.1000', 'tmpl_id':1, 'mobile':'13346655259'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': ip非法}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring': ip非法}" != "{'retCode': 0, 'retInfo': 'ok'}"
- {'faultcode': soap:Server, 'faultstring': ip非法}
+ {'retCode': 0, 'retInfo': 'ok'}


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_47' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_47: tmpl_id为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_47' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_47'>错误</button>
    <div id='div_ft1_47' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_47' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_47'>错误</button>
    <div id='div_ft1_47' class="collapse in" align="left">
    <pre>
    
ft1_47: 请求数据: {'client_ip':'77.80.57.15', 'tmpl_id':'', 'mobile':'13330257357'}
&lt;suds.sax.document.Document object at 0x000001DFC53CDE80&gt;
请求数据--&gt; {'client_ip':'77.80.57.15', 'tmpl_id':'', 'mobile':'13330257357'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}
服务器响应数据--&gt; {'faultcode': soap:Client, 'faultstring': Unmarshalling Error: For input string: "" }
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring': 短信模板配置错误}" != '{\'faultcode\': soap:Client, \'faultstri[45 chars]"" }'
- {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}
+ {'faultcode': soap:Client, 'faultstring': Unmarshalling Error: For input string: "" }


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_48' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_48: tmpl_id为数字</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_48' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_48'>错误</button>
    <div id='div_ft1_48' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_48' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_48'>错误</button>
    <div id='div_ft1_48' class="collapse in" align="left">
    <pre>
    
ft1_48: 请求数据: {'client_ip':'24.150.52.170', 'tmpl_id':1, 'mobile':'13319298321'}
请求数据--&gt; {'client_ip':'24.150.52.170', 'tmpl_id':1, 'mobile':'13319298321'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring': 短信模板配置错误}" != "{'retCode': 0, 'retInfo': 'ok'}"
- {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}
+ {'retCode': 0, 'retInfo': 'ok'}


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_49' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_49: tmpl_id为非1（2）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_49' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_49'>通过</button>
    <div id='div_pt1_49' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_49' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_49'>通过</button>
    <div id='div_pt1_49' class="collapse in" align="left">
    <pre>
    
pt1_49: 请求数据: {'client_ip':'241.15.76.179', 'tmpl_id':'2', 'mobile':'13331495914'}
&lt;suds.sax.document.Document object at 0x000001DFC57B8320&gt;
请求数据--&gt; {'client_ip':'241.15.76.179', 'tmpl_id':'2', 'mobile':'13331495914'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 短信模板配置错误}


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_50' class='none'>
    <td class='errorCase'><div class='testcase'>test_send_m_code_50: tmpl_id为非1（0）</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_50' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_50'>错误</button>
    <div id='div_ft1_50' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_50' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_50'>错误</button>
    <div id='div_ft1_50' class="collapse in" align="left">
    <pre>
    
ft1_50: 请求数据: {'client_ip':'13.139.90.242', 'tmpl_id':'0', 'mobile':'13306503783'}
&lt;suds.sax.document.Document object at 0x000001DFC5056B38&gt;
请求数据--&gt; {'client_ip':'13.139.90.242', 'tmpl_id':'0', 'mobile':'13306503783'}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring':'短信模板不存在}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 短信模板不存在}
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 88, in test_send_m_code
    self.assertEqual(case.expected_data, str(response))
  File "C:\Python37\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python37\lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Python37\lib\unittest\case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: "{'faultcode': soap:Server, 'faultstring':'短信模板不存在}" != "{'faultcode': soap:Server, 'faultstring': 短信模板不存在}"
- {'faultcode': soap:Server, 'faultstring':'短信模板不存在}
?                                          ^
+ {'faultcode': soap:Server, 'faultstring': 短信模板不存在}
?                                          ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\api_webservice\library\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\keen\PycharmProjects\api_webservice\testcases\test_sendMCode.py", line 92, in test_send_m_code
    self.wb.write_data(row=self.row, column=9, msg=result, fill=PatternFill(fgColor="FF00FF"))
TypeError: write_data() got an unexpected keyword argument 'fill'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_51' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_51: 手机号为数字</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_51' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_51'>通过</button>
    <div id='div_pt1_51' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_51' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_51'>通过</button>
    <div id='div_pt1_51' class="collapse in" align="left">
    <pre>
    
pt1_51: 请求数据: {'client_ip':'125.238.205.152', 'tmpl_id':'1', 'mobile':'13325043030'}
请求数据--&gt; {'client_ip':'125.238.205.152', 'tmpl_id':'1', 'mobile':'13325043030'}
期望结果---&gt; {'retCode': '0', 'retInfo': 'ok'}
服务器响应数据--&gt; {'retCode': 0, 'retInfo': 'ok'}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_52' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_52: 10位手机号</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_52' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_52'>通过</button>
    <div id='div_pt1_52' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_52' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_52'>通过</button>
    <div id='div_pt1_52' class="collapse in" align="left">
    <pre>
    
pt1_52: 请求数据: {'client_ip':'195.76.154.103', 'tmpl_id':'1', 'mobile':1557328489}
&lt;suds.sax.document.Document object at 0x000001DFC5820A58&gt;
请求数据--&gt; {'client_ip':'195.76.154.103', 'tmpl_id':'1', 'mobile':1557328489}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_53' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_send_m_code_53: 12位手机号</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_53' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_53'>通过</button>
    <div id='div_pt1_53' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_53' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_53'>通过</button>
    <div id='div_pt1_53' class="collapse in" align="left">
    <pre>
    
pt1_53: 请求数据: {'client_ip':'13.205.117.205', 'tmpl_id':'1', 'mobile':155732848911}
&lt;suds.sax.document.Document object at 0x000001DFC507D278&gt;
请求数据--&gt; {'client_ip':'13.205.117.205', 'tmpl_id':'1', 'mobile':155732848911}
期望结果---&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}
服务器响应数据--&gt; {'faultcode': soap:Server, 'faultstring': 手机号码错误}


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>53</td>
    <td>39</td>
    <td>0</td>
    <td>14</td>
    <td>通过率：73.58%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
